src/window/win.cc

```diff
@@ -947,12 +949,17 @@ namespace SSC {
                             if (fs::exists(path)) {
                               String headers;
                               char* body;
+                              String moduleSource;
 
-                              auto moduleUri = "file://" + replace(path.string(), "\\\\", "/");
-                              auto moduleSource = trim(tmpl(
-                                moduleTemplate,
-                                Map { {"url", String(moduleUri)} }
-                              ));
+                              if (uri_s.compare("preload") == 0) {
```

Comment:
what does this URI look like?

Reply:
The original value is:
`socket://preload/`
By the time it gets here, `socket://` and the trailing `/` have been removed.